src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String INDENTATION = "    ";
+    public static final String LINE = INDENTATION
+            + "____________________________________________________________";
+    public static final int MAX_NUM = 100;
+
+    public static Task[] tasks = new Task[MAX_NUM];
+    public static int count = 0;
+    public static Scanner in = new Scanner(System.in);
+
     public static void main(String[] args) {
+        printGreetMsg();
+        String userInput = in.nextLine();
+        processUserInput(userInput);
+        printByeMsg();
```

Comment:
I think the names you are using for your methods are good and intuitive!

Reply:
Thank you!